scripts/pylint_extensions.py

```diff
@@ -794,6 +795,40 @@ def visit_functiondef(self, node):
             self.add_message('function-args-order-cls', node=node)
 
 
+class SingleCharAndNewlineAtEOFChecker(checkers.BaseChecker):
+    """Checker for single character files and newline at EOF."""
+
+    __implements__ = interfaces.IRawChecker
+    name = 'newline-at-eof'
+    priority = -1
+    msgs = {
+        'C0007': (
+            'Files should end in a single newline character.',
+            'newline-at-eof',
+            'Please enter a newline at the end of the file'),
+        'C0008': (
+            'Only one character in file',
+            'only-one-character',
+            'Files with only one character are not allowed'),
+    }
+
+    def process_module(self, node):
+        """Process a module.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node to access module content.
+        """
+
+        file_content = node.stream().readlines()
+        file_length = len(file_content)
+        print '@@@@@@@'
+        print len(file_content[0])
```

Comment:
None of the pylint checks are running since this line errors out:
```
Process Process-4:
Traceback (most recent call last):
  File "/opt/python/2.7.14/lib/python2.7/multiprocessing/process.py", line 267, in _bootstrap
    self.run()
  File "/opt/python/2.7.14/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "scripts/pre_commit_linter.py", line 618, in _lint_py_files
    exit=False).linter
  File "/home/travis/build/oppia/oppia_tools/pylint-1.9.3/pylint/lint.py", line 1353, in __init__
    linter.check(args)
  File "/home/travis/build/oppia/oppia_tools/pylint-1.9.3/pylint/lint.py", line 774, in check
    self._do_check(files_or_modules)
  File "/home/travis/build/oppia/oppia_tools/pylint-1.9.3/pylint/lint.py", line 907, in _do_check
    self.check_astroid_module(ast_node, walker, rawcheckers, tokencheckers)
  File "/home/travis/build/oppia/oppia_tools/pylint-1.9.3/pylint/lint.py", line 982, in check_astroid_module
    checker.process_module(ast_node)
  File "/home/travis/build/oppia/oppia/scripts/pylint_extensions.py", line 825, in process_module
    print len(file_content[0])
IndexError: list index out of range
```

Reply:
It was cause by `__init__.py` file; fixed it!